pkg/get/download.go

```diff
@@ -40,22 +40,11 @@ func Download(tool *Tool, arch, operatingSystem, version string, downloadMode in
 		}
 
 		outFilePathDir := filepath.Dir(outFilePath)
-		if len(tool.BinaryTemplate) > 0 {
-			fileName, err := GetBinaryName(tool, strings.ToLower(operatingSystem), strings.ToLower(arch), version)
-			if err != nil {
-				return "", "", err
-			}
-			outFilePath = path.Join(outFilePathDir, fileName)
-		} else {
-			outFilePath = path.Join(outFilePathDir, tool.Name)
-		}
 
-		if strings.Contains(strings.ToLower(operatingSystem), "mingw") && tool.NoExtension == false {
-			outFilePath += ".exe"
-		}
-
-		if strings.HasSuffix(downloadURL, "tar.gz") || strings.HasSuffix(downloadURL, "tgz") {
-			untarErr := archive.Untar(archiveFile, outFilePathDir)
+		var tarFileName string
+		if strings.HasSuffix(downloadURL, "gz") || strings.HasSuffix(downloadURL, "tar.gz") || strings.HasSuffix(downloadURL, "tgz") {
+			var untarErr error
+			tarFileName, untarErr = archive.Untar(archiveFile, outFilePathDir)
```

Comment:
Unfortunately, there can often be more than one file within an archive. Helm is an example of that, and at times there is also a README and license file.

Can you explain your approach here, or your algorithm for determining the binary name?

Reply:
Do you have an arkade example for me to test the helm case?

Reply:
I suggest that you look at the tests to see which filenames mention an archive like .zip or .tar.gz

You also didn't answer my question yet:

> Can you explain your approach here, or your algorithm for determining the binary name?

Alex